### ISSUE TITLE
manage.py poll_rss fails.

### ISSUE THREAD

**Issue by COLABORATI** [stephenmcd/drum#38]
_sigh_... next step in README, next error... 
```
ValueError: <Rating: Rating object> instance isn't saved. Use bulk=False or save the object first.
```

**Comment by stephenmcd** [comment]
Already fixed in 7b327615a93ee4fa04d595b53d20f9d9f34db925

Since you're hitting a lot of these, suggest verifying the development version (and dev dependencies) before opening an issue.

**Comment by COLABORATI** [comment]
Ah, ok you mean this is fixed in the repo, but not yet on pypi? Or am I misunderstanding that?
